components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -452,4 +453,52 @@ - (void)testTitlesNumberOfLinesPassedToViewsAfterItemsAssigned {
   }
 }
 
+- (void)testItemForPointInsideItemView {
```

Comment:
Our test names describe the full expectations of the test using a format of "test that &lt;name of the test>". This makes it easier to skim the test names in order to understand what the expected behavior is.

E.g. "test that tabBarItemForPoint inside an item view returns the item for that view" might be written as "testTabBarItemForPointInsideItemViewReturnsItemForThatView"

As is presently written, this test name leaves the reader wondering what the expectation of "items for point inside item view" might be. It does describe what the test is about, but not what the test's expectation is.

Reply:
@jverkoey This is a good goal to strive for, but it does not accurately describe all of our tests. It's also not a team norm that I'm aware of.  I think it's worth bringing your suggestion to the team generally, but shouldn't remain a blocker for this PR.  

Naming nits are always tough, but I think plenty of our tests explain the "when" and not the "then".  For example, in this same component I wrote the unit test `testSizeToFitNilLabel` and not `testSizeToFitWithNilLabelReturnsCGRectZero`.

Reply:
Took a stab at the method names.  It seemed like an easy enough change to make.